### ISSUE TITLE
Make Postgres Loader Collapse Duplicates into Single Column

### ISSUE THREAD

**Issue by bsmithgall** [codeforamerica/w-drive-extractor#1]
### Current Status

```
select * from company where company = 'CILENTI CONSTRUCTION CO.';

w_drive-# ;
 company_id |         company          | bus_type
------------+--------------------------+----------
         91 | CILENTI CONSTRUCTION CO. |
         92 | CILENTI CONSTRUCTION CO. |
         93 | CILENTI CONSTRUCTION CO. |
         96 | CILENTI CONSTRUCTION CO. |
         97 | CILENTI CONSTRUCTION CO. |
(5 rows)
```

This is also causing problems when you are linking tables together.
### What we want

Solution is to collapse those five lines into one. This solution should also update all the relevant relationships as well.

**Comment by bsmithgall** [comment]
Steps to possible solution (big thanks to JEK)
1. Build all data as normal
2. Since we know that all of the indices are going to match across tables, we want to hash (or something) all of the duplicate rows in the tables that have `from_relations` (TODO: Add the ability to dedupe more tables!)
3. If we find duplicates, store them in a dictionary, where the key is the id/index of that row, and the value is the value of the hash
4. Iterate through the table rows, adding a new column with the hash from the dictionary described in step 3.
5. Go through the `to_relations` table, and add a new column as described above
6. Delete the old id column
7. Take a `set` of the remainder
8. ????
9. Profit

**Comment by bsmithgall** [comment]
Fixed by #3
